lib/miq_expression/field.rb

```diff
@@ -40,6 +41,10 @@ def string?
     column_type == :string
   end
 
+  def numeric?
+    [:fixnum, :integer, :Fixnum, :float].include?(column_type)
```

Comment:
Could we add tests for this? I was surprised that `:fixnum` and `:Fixnum` were necessary